soc/arm/atmel_sam/same70/dts_fixup.h

```diff
@@ -8,22 +8,7 @@
 
 /* SoC level DTS fixup file */
 
-#define DT_SPI_0_BASE_ADDRESS	DT_ATMEL_SAM_SPI_40008000_BASE_ADDRESS
-#define DT_SPI_0_NAME		DT_ATMEL_SAM_SPI_40008000_LABEL
-#define DT_SPI_0_IRQ		DT_ATMEL_SAM_SPI_40008000_IRQ_0
-#define DT_SPI_0_IRQ_PRI		DT_ATMEL_SAM_SPI_40008000_IRQ_0_PRIORITY
-#define DT_SPI_0_PERIPHERAL_ID	DT_ATMEL_SAM_SPI_40008000_PERIPHERAL_ID
-
-#define DT_SPI_1_BASE_ADDRESS	DT_ATMEL_SAM_SPI_40058000_BASE_ADDRESS
-#define DT_SPI_1_NAME		DT_ATMEL_SAM_SPI_40058000_LABEL
-#define DT_SPI_1_IRQ		DT_ATMEL_SAM_SPI_40058000_IRQ_0
-#define DT_SPI_1_IRQ_PRI		DT_ATMEL_SAM_SPI_40058000_IRQ_0_PRIORITY
-#define DT_SPI_1_PERIPHERAL_ID	DT_ATMEL_SAM_SPI_40058000_PERIPHERAL_ID
-
-#define DT_ADC_0_NAME	DT_ATMEL_SAM_AFEC_4003C000_LABEL
-#define DT_ADC_1_NAME	DT_ATMEL_SAM_AFEC_40064000_LABEL
-
-#define DT_FLASH_DEV_BASE_ADDRESS		DT_ATMEL_SAM_FLASH_CONTROLLER_400E0C00_BASE_ADDRESS
-#define DT_FLASH_DEV_NAME			DT_ATMEL_SAM_FLASH_CONTROLLER_400E0C00_LABEL
+#define DT_ADC_0_NAME	DT_LABEL(DT_NODELABEL(afec0))
+#define DT_ADC_1_NAME	DT_LABEL(DT_NODELABEL(afec1))
```

Comment:
@galak Is there an upcoming fix to get rid of these? (e.g. converting these to `chosen` or something like that)

Reply:
I don't have a plan yet.  `chosen` doesn't really solve the issue of how these are used by the shell.